kubectl/kubectl.go

```diff
@@ -28,6 +31,15 @@ func AppsClient() (appsv1.AppsV1Interface, error) {
 	return client.AppsV1(), nil
 }
 
+//HPAClient is used for interacting with horizontal pod autoscalers
+func HPAClient() (autoscalingv1.AutoscalingV1Interface, error) {
+	_, client, err := getKubeClient()
```

Comment:
does this act as a pattern match assignment if getKubeClient() returns a tuple of 3 elements?

Reply:
Go can return multiple values from a single function. This is just asking to ignore element 0 as we've got no use for it within this context. It's grabbing the client, and any errors generated when creating the client; If there are no errors, we're good to go and can use the client.